src/snmp/Forwarder.cc

```diff
@@ -89,7 +89,7 @@ Snmp::Forwarder::sendError(int error)
 void
 Snmp::SendResponse(unsigned int requestId, const Pdu& pdu)
 {
-    debugs(49, 5, HERE);
+    debugs(49, 5, "");
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.